Tests/Common/Statistics/AnnualPerformanceTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+using System.Linq;
+using System.Collections.Generic;
+using NUnit.Framework;
+using QuantConnect.Util;
+using System;
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Tests.Common.Statistics
+{
+    [TestFixture]
+    public class AnnualPerformanceTests
+    {
+        private List<TradeBar> _spy = new List<TradeBar>();
+
+        [SetUp]
```

Comment:
Think we could use `TestFixtureSetUp` here so that the setup happens once for all the fixture and not once per test, for teardown it would be `TestFixtureTearDown`. Same for `TrackingErrorTests`

Nice tests!